### DISCUSSION TITLE
Generate documents out of Netbox content

### DISCUSSION THREAD

**Discussion by blindzero** [netbox-community/netbox#16026] [Help Wanted!]
Hi everybody,

assuming the following: you are in a regulated environment...automation, documentation, standardization, all fine...
_But_....what if worst case scenario or "s**t hits the fan": everything is really down! No network, your clusters are screwed up...whatever: you cannot access your Netbox anymore!

In old times you had paper-style offline files in a shelf with your documentation...with SOPs, the latest status and so on.
With tools like Netbox you don't wanna maintain this "worst-case-only" documents anymore manually.

Who has such a scenario...how do you solve it? And please, don't tell me to remove all SPoFs, there were plent of outages in the internet / the cloud that show us: no matter how much money you throw into an redundant environment: it always can happen.

Thanks a lot!
Matthias

**Comment by markkuleinio** [comment]
My DR plan for NetBox is to have the daily database backup available somewhere, maybe also the configuration file (if it has specific essential customizations like custom status values). I can then rebuild a NetBox server somewhere in minutes and import the database there.

Obviously that will not bring up any automation or integration with other systems, but this can be compared to the mentioned paper files.

If your NetBox data relies on custom models provided by plugins, you also need to be sure to have the plugin installers (correct versions) available as well.

**Comment by blindzero** [comment]
Hi @markkuleinio,
thanks for your input. That is fine for the system DR itself, but I guess not for worst-case incident compliance...
E.g. power circuits documentations are in Netbox... power issue --> no backup system (at least as soon as batteries are out) ;-)
It would be so great to have a PDF out of the stuff....unfortunately I am manager not developer :-D

**Reply by markkuleinio** [reply]
> I am manager not developer :-D

Oh then it's easy: you just hire or order a developer to write your company the "NetBox to XLSX/PDF" plugin/app that fulfills your business requirement, right?

**Reply by blindzero** [reply]
Well...wanted to avoid to re-invent the wheel...
Couldn't imagine that there is no solution out there...

**Comment by Kage1** [comment]
My suggestion would be to export the critical data to CSV, import that into Excel, print, then bind. Depending on how often your critical info updates would inform you how often you'd need to update your offline copy.
